docs/src/site/twiki/FalconDocumentation.twiki

```diff
@@ -249,9 +250,10 @@ entity from the falcon configuration store. Delete operation on an entity would
 no dependent entities on the deleted entity.
 
 ---+++ Update
-Update operation allows an already submitted/scheduled entity to be updated. Cluster update is currently
-not allowed. Feed update can cause cascading update to all the processes already scheduled. Process update triggers
-update in falcon if entity is updated. The following set of actions are performed in scheduler to realize an update:
+Update operation allows an already submitted/scheduled entity to be updated. Cluster update will require user to  update
+ dependent Feed and Process entities that are already scheduled. Feed update can cause cascading update to all the processes
+already scheduled. Process update triggers update in falcon if entity is scheduled. The following set of actions are
```

Comment:
Better to split cluster update and feed/process update to two paragraphs, one for each.